README.md

```diff
@@ -104,8 +105,8 @@ The `Antenna` object contains the following name/value pairs:
 |`low_frequency`|false|float|Hz|Low frequency of operational range.|
 |`high_frequency`|false|float|Hz|High frequency of operational range.|
 |`gain`|false|float|dBi|Antenna gain in direction of maximum radiation or reception.|
-|`horizontal_gain_pattern`|false|array|dBi|Antenna gain pattern in horizontal plane.|
-|`vertical_gain_pattern`|false|array|dBi|Antenna gain pattern in vertical plane.|
+|`horizontal_gain_pattern`|false|array of floats|dBi|Antenna gain pattern in horizontal plane from 0 to 359 degrees in 1 degree steps.|
+|`vertical_gain_pattern`|false|array of floats|dBi|Antenna gain pattern in vertical plane from -90 to +90 degrees in 1 degree steps.|
```

Comment:
Do we need an array if we specify in the description the start, end, and increment? List?